artwork/linedrawings/aaron.jsx

```diff
@@ -0,0 +1,182 @@
+import React from "react";
+
+function Icon() {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      width="592.68"
+      height="435.138"
+      version="1.1"
+      viewBox="0 0 156.813 115.13"
+    >
+      <g transform="translate(-124.226 -143.605)">
+        <path
+          fill="none"
+          stroke="#000"

```

Comment:
The only color you can use is `currentColor`. So replace `#000` with `currentColor.